src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of
+ * related CSS properties (see an example
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringCleared = sourceString.replace(/ /g, '').split('\n').join('');
+  const finalString = stringCleared
+    .replace(/1pxsolid#e8e8e8/g, '1px solid #e8e8e8')
+    .replace(/left!important/g, 'left !important')
+    .replace(/all.2sease-in-out/g, 'all .2s ease-in-out');
```

Comment:
you shouldn't rely on initial data in you logic
This is hardcode  - need to remove it